CHANGELOG.md

```diff
@@ -1,21 +1,14 @@
-# NEXT RELEASE
+# 2.3.2 Release notes
 
 ### Bugfixes
 * Fixed bug when encryption was used. It would cause crashes and corrupted data 
   (sometimes pieces of data from earlier commits could be seen). PR #2465.
+  (fixes https://github.com/realm/realm-core/issues/2383)
```

Comment:
Just a minor comment for next time. The purpose of this PR is to bump the version (and prepare for release), so this change should (ideally) not be here. It's all with good intend, of course.